etc/bash_completion.d/stratis

```diff
@@ -1,17 +1,21 @@
 _stratis()
 {
-	local cur prev x2prev x3prev opts root_subcommands pool_subcommands fs_subcommands blockdev_subcommands daemon_subcommands complete_pools
+	local cur prev x2prev x3prev opts root_subcommands pool_subcommands fs_subcommands blockdev_subcommands daemon_subcommands complete_pools blockdevs first second third
 	COMPREPLY=()
 	cur="${COMP_WORDS[COMP_CWORD]}"
 	prev="${COMP_WORDS[COMP_CWORD-1]}"
 	x2prev="${COMP_WORDS[COMP_CWORD-2]}"
 	x3prev="${COMP_WORDS[COMP_CWORD-3]}"
+	first="${COMP_WORDS[1]}"
+	second="${COMP_WORDS[2]}"
+	third="${COMP_WORDS[3]}"
 	opts="-h --help --version"
 	root_subcommands="--help --version daemon pool blockdev filesystem"
 	pool_subcommands="create list rename destroy"
 	fs_subcommands="create snapshot list rename destroy"
 	blockdev_subcommands="add list"
 	daemon_subcommands="redundancy version"
+	blockdevs=$(ls -l /dev | awk '{printf("/dev/%s\n",$10)}' | tr '\n' ' ')
```

Comment:
Looks good to me. The only question I have (and please understand I am not very familiar with bash) is if this handles blockdev completions down into dirs under dev e.g. /dev/disk/by-id/ or not?

Reply:
That doesn't do it, I changed it and now I removed the useless awk (I found out echo actually does what I needed already) and now it also looks in subdirs and subdirs of subdirs so that it finds devices under /dev/disk/by-X/